test/data/doubles_multiline_string.py

```diff
@@ -1,7 +1,7 @@
-""" This "should"
+s = """ This "should"
```

Comment:
Is this considered a docstring by the AST? If not, I'd consider this a regression by our detector =/